include/crypto_config_structs.h

```diff
@@ -89,6 +89,12 @@ typedef enum
     SA_INCREMENT_NONTRANSMITTED_IV_TRUE
 } SaIncrementNonTransmittedIvPortion;
 
+typedef enum
+{
+    KMC_PADDING,
+    LGC_PADDING
+} padding_control;
+
```

Comment:
See my config comment above -- padding_control can be deleted throughout the code.

Reply:
Removed